zilencer/management/commands/populate_db.py

```diff
@@ -134,6 +134,12 @@ def add_arguments(self, parser: CommandParser) -> None:
                             default=0,
                             help='The number of extra streams to create')
 
+        parser.add_argument('--max-topics',
+                            dest='max_topics',
+                            type=int,
+                            default=0,
```

Comment:
I think default=None is more appropriate here; though more work would be needed to support it, in theory `0` should be a valid value here.